python/lsst/qa/explorer/parquetTable.py

```diff
@@ -0,0 +1,268 @@
+"""
+Implementation of thin wrappers to pyarrow.ParquetFile.
+"""
+import re
+import json
+from itertools import product
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+
+class ParquetTable(object):
+    """Thin wrapper to pyarrow's ParquetFile object
+
+    Call `to_df` method to get a `pandas.DataFrame` object,
+    optionally passing specific columns.
+
+    Parameters
+    ----------
+    filename : str
+        Path to Parquet file.
+
+    """
+    def __init__(self, filename=None, dataFrame=None):
+        if filename is not None:
+            self._pf = pq.ParquetFile(filename)
+            self._df = None
+            self._pandas_md = None
+        elif dataFrame is not None:
+            self._df = dataFrame
+            self._pf = None
+        else:
+            raise ValueError('Either filename or dataFrame must be passed.')
+
+        self._columns = None
+        self._columnIndex = None  
+
+    @property
```

Comment:
Should these be exposed as public properties?  Isn't `to_df()` the expected interface?